### DISCUSSION TITLE
Add BYPS=1, FRNT=1, TUSB=1, and HSTP=x,z,size to gopro hero 9

### DISCUSSION THREAD

**Discussion by qsffffff** [gopro/labs#515] [Feature Requests]
A lot of fpv pilots still use gopro hero 9 as a naked camera so we want to be able to control everything from the front screen and these commands are so useful!
Thanks!

**Comment by dnewman-gpsw** [comment]
Same for HERO10 and 11, and likely 12.  Is there a question?

**Reply by qsffffff** [reply]
No, i asked for these features to be added to gopro hero 9 as they are only for 10 11 and 11mini
